app/models/authentication_configuration_script_base.rb

```diff
@@ -0,0 +1,4 @@
+class AuthenticationConfigurationScriptBase < ApplicationRecord
+  belongs_to :authentication
+  belongs_to :configuration_script_base, :foreign_key => 'configuration_script_id'
```

Comment:
Now you have to tell me because I just deleted the spec that would tell me the answer 😱